Meadowlark-js/packages/meadowlark-plugin-dynamodb-opensearch/src/types/AwsSdkLibDynamoDb.ts

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Is this old code that just moved to a new file?

Reply:
It's a temporary duplication due to an existing leaky abstraction. The AWS type `PutItemInputAttributeMap` is in the meadowlark model object ForeignKeyItem. meadowlark-plugin-dynamodb-opensearch is the correct place for this file. I reduced the duplicate in meadowlark-core to just define `PutItemInputAttributeMap`. Later removal will be obvious when the time comes, because it currently requires meadowlark-core to depend on AWS libraries.